### PR TITLE
Use the generic API to report ept_violation events

### PR DESCRIPTION
**Pull request by bbosinta**

### CODE REVIEW

File: sources/vbh_rt.c [hunk 1]
```diff
@@ -182,13 +181,15 @@ void handle_ept_violation(struct vcpu_vmx *vcpu)
 
 	printk("EPT_VIOLATION at GPA -> 0x%llx GVA -> 0x%llx, exit_qulification = 0x%lx, G_RSP = 0x%lx, G_RIP=0x%lx\n", gpa, gla, exit_qual, g_rsp, g_rip);
 
-	if (hvi_invoke_ept_violation_handler(gpa, gla, &allow)) {
-		printk(KERN_ERR "vmx-root: hvi_invoke_ept_violation_handler failed\n");
+	if (hvi_handle_ept_violation(gpa, gla, &allow)) {
```
Comment (sainath14) [inline sources/vbh_rt.c:184]: @bbosinta 
Is "allow == true" a possibility. If yes, then we have to remove the comment in else if (allow) and just return.

**Review by sainath14 — COMMENTED** [review]
Is "allow == true" a possibility. If yes, then we have to remove the comment in else if (allow) and just return.

File: sources/vbh_rt.c (hunk 1 above)
Comment (bbosinta) [reply sources/vbh_rt.c:184]: @sainath14 
Yes. When the APIs required by the single step logic will be implemented hvi will take care of eventual false positives. But for now i think it is better to skip the instruction regardless the value of allow. 
Is it ok if i remove the whole "else if..." body (including the return statement)?

Comment (sainath14) [reply sources/vbh_rt.c:184]: Yes. That makes it better. Please add some comments stating handling "allow == true" is a to-do.

**Pull request merged**